src/Microsoft.DotNet.Interactive.CSharp/InteractiveWorkspace.cs

```diff
@@ -59,6 +119,16 @@ private Solution RemoveWorkingProjectFromSolution(Solution solution)
             return solution;
         }
 
+        private IEnumerable<MetadataReference> GetReferenceSet(Compilation compilation)
```

Comment:
Since you're `ToArray`ing this you might as well return `IReadOnlyList<MetadataReference>`.

Reply:
Done.